docs/vercel/package.json

```
@@ -7,12 +7,17 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "check": "svelte-check --tsconfig ./tsconfig.json"
+    "check": "check:svelte && check:prettier && check:eslint",
+    "check:svelte": "svelte-check --tsconfig ./tsconfig.json",
+    "check:prettier": "prettier --check 'src/**/*.{ts,js,svelte}'",
```

```suggestion
    "check:prettier": "prettier --check **/*.{ts,js,svelte,css,html,json}",
```
assuming it ignores the `node_modules`.

---

Yeah, [`node_modules` is ignored by default](https://prettier.io/docs/en/cli.html#--with-node-modules).

Good call, I've always ESLint way more than Prettier so I forgot that it also works for the non-programming languages, too :laughing: :facepalm: